src/main/java/com/report/portal/business/steps/DashboardsSteps.java

```diff
@@ -11,16 +11,16 @@ public class DashboardsSteps {
     private final AllDashboardsPage allDashboardsPage = new AllDashboardsPage();
     private final DashboardPage dashboardPage = new DashboardPage();
 
-    public static String createWidgetName(WidgetType widgetType, int randomValue) {
-        return widgetType.getWidgetTitle().concat("_" + randomValue);
-
-    }
-
     public DashboardsSteps selectDashboard(String dashboardName) {
         allDashboardsPage.goToDashboard(dashboardName);
         return this;
     }
 
+    public DashboardsSteps assertDashboardVisible(String dashboardName) {
```

Comment:
I would like to reccomend you to rename "assertDashboardVisible" to "checkThatDashboardVisible". Because you don't have assert here.